docs/source/technical_tutorials/user_data/stac_metadata.rst

```diff
@@ -0,0 +1,174 @@
+MAAP STAC Metadata Recommendations
+==================================
+
+MAAP uses the STAC (SpatioTemporal Asset Catalog) specification, which
+provides a common structure for describing and cataloging geospatial
+data `(STAC) <https://stacspec.org/en>`__.
+
+At a minimum, MAAP metadata should conform to STAC specification
+requirements. MAAP also requires the STAC-equivalent of certain `Unified
+Metadata Model
+(UMM) <https://www.earthdata.nasa.gov/about/esdis/eosdis/cmr/umm>`__
+metadata elements (UMM is a metadata schema used in `NASA’s Common
+Metadata Repository
+(CMR) <https://www.earthdata.nasa.gov/about/esdis/eosdis/cmr>`__). These
+elements are included below, and are broken down at the collection and
+item level.
+
+**Note**: NASA CMR also uses Global Change Master Directory (GCMD)
+Keywords to help with consistency and searchability. As a best practice,
+MAAP recommends using GCMD keywords in applicable fields, which are
+noted below. For more information, see `“GCMD Keyword
+Viewer” <https://www.earthdata.nasa.gov/data/tools/idn/gcmd-keyword-viewer>`__.
+Downloadable CSV files of GCMD keywords are also provided for relevant
+fields.
+
+STAC Collection-Level Metadata Fields
+-------------------------------------
+
+-  ``id``: **Required**. The unique identifier of the collection.
+-  ``version``: **Required**. The version of the data. This is different
+   than the ``stac_version`` element.
+-  ``title``: **Required**. A short descriptive title of the datatset.
+   This should differ from the ``id``.
+-  ``sci:doi``: **Required if available**. The DOI of the collection
```

Comment:
We should ask users to include other fields from the scientific extension if `sci:doi` is not available. Using `sci:citation` like we did for icesat2-boreal where the publication was still in prep.

Reply:
@hrodmn Yes I just wanted to be clear what is STAC required (that a Validator might catch), and what is MAAP asking for in addition to that to ensure good quality metadata.